nextcord/guild.py

```diff
@@ -3457,3 +3461,223 @@ async def register_application_commands(self, *commands: BaseApplicationCommand)
+
+        Retrieves an auto moderation rule from Discord.
+
+        You must have the :attr:`~Permissions.manage_guild` to be able to do this.
+
+        Parameters
+        -----------
+        rule_id: :class:`int`
+            The rule ID of the auto moderation rule you want to fetch.
+
+        Returns
+        --------
+        :class:`AutoModerationRule`
+             The rule from the rule ID.
+
+        Raises
+        -------
+        Forbidden
+            You do not have proper permissions to do this.
+        NotFound
+            The rule you tried to find doesn't exist.
+        """
+        rule = await self._state.http.get_automod_rule(guild_id=self.id, rule_id=rule_id)
+        return AutoModerationRule(guild=self, state=self._state, data=rule)
+
+    def get_automod_rule(self, rule_id: int) -> Optional[AutoModerationRule]:
+        """Get an auto moderation rule.
+
+        Parameters
+        ----------
+        rule_id: :class:`int`
+            The rule ID to find for.
+
+        Returns
+        -------
+        Optional[:class:`AutoModerationRule`] or ``None`` if not found.
+        """
+        return self._state.get_automod_rule(id=rule_id)
+
+    @overload
+    async def create_automod_rule(
+        self,
+        *,
+        name: str,
+        event_type: EventType = EventType.message_send,
+        trigger_type: TriggerType,
+        keyword_filters: List[str] = MISSING,
+        notify_channel: GuildChannel = MISSING,
+        enabled: bool = True,
+        exempt_roles: List[Role] = MISSING,
+        exempt_channels: List[GuildChannel] = MISSING,
+    ) -> AutoModerationRule:
+        ...
+
+    @overload
+    async def create_automod_rule(
+        self,
+        *,
+        name: str,
+        event_type: EventType = EventType.message_send,
+        trigger_type: TriggerType,
+        keyword_filters: List[str] = MISSING,
+        timeout_seconds: int = MISSING,
+        enabled: bool = True,
+        exempt_roles: List[Role] = MISSING,
+        exempt_channels: List[GuildChannel] = MISSING,
+    ) -> AutoModerationRule:
+        ...
+
+    @overload
+    async def create_automod_rule(
+        self,
+        *,
+        name: str,
+        event_type: EventType = EventType.message_send,
+        trigger_type: TriggerType,
+        presets: List[KeywordPresetType] = MISSING,
+        notify_channel: GuildChannel = MISSING,
+        enabled: bool = True,
+        exempt_roles: List[Role] = MISSING,
+        exempt_channels: List[GuildChannel] = MISSING,
+    ) -> AutoModerationRule:
+        ...
+
+    @overload
+    async def create_automod_rule(
+        self,
+        *,
+        name: str,
+        event_type: EventType = EventType.message_send,
+        trigger_type: TriggerType,
+        presets: List[KeywordPresetType] = MISSING,
+        notify_channel: GuildChannel = MISSING,
+        enabled: bool = True,
+        exempt_channels: List[GuildChannel] = MISSING,
+    ) -> AutoModerationRule:
+        ...
```

Comment:
Is there a reason to have this override without `exempt_roles`?

I'm thinking maybe you meant to add `timeout_seconds`?

Reply:
```suggestion
    @overload
    async def create_automod_rule(
        self,
        *,
        name: str,
        event_type: EventType = EventType.message_send,
        trigger_type: TriggerType,
        presets: List[KeywordPresetType] = MISSING,
        notify_channel: GuildChannel = MISSING,
        timeout_seconds: int = MISSING,
        enabled: bool = True,
        exempt_roles: List[Role] = MISSING,
        exempt_channels: List[GuildChannel] = MISSING,
    ) -> AutoModerationRule:
        ...
```